### ISSUE TITLE
New password protection option

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1305]
My understanding of the new password protection option in the access settings is that it still requires users to go to a different url (pwplay) instead of simply play. I understand why this was originally set up but I think that now this functionality is being surfaced to more people it would make much more sense for the play URL to be used for all the access settings. This would be beneficial for several reasons:
- Consistency - at the moment we use play for password protected and public projects even though they act differently when people visit them
- If authors want to change the access setting (e.g. go from pwplay password option to public) they would have to distribute a URL again as the original one would no longer work. At the moment, if you change between different access options the URL is always the same so you can change settings and not worry what URLs have been sent to people or linked to from other websites.
- If we had the new custom password protection option working with play instead of pwplay then we could leave the old way of doing pwplay with the peer review password working for anyone that has stumbled across this and started using it

Thoughts?

**Comment by FayCross** [comment]
@torinfo I think tis has broken. If I select the password protected (play) option now the normal password protected radio button gets selected - I can't get it to select the play version.

**Comment by FayCross** [comment]
@torinfo - I've fixed the issue I showed you yesterday where a click on the password protected (play) label selected the incorrect radio box. I've also tweaked the wording of the options. I've found a new problem though - if you have the new password option selected and enter the password then it all seems to work ok - however, any standalone pages that open require you to re-enter the password. Is there a way around this?
